manifests/e/Egnyte/EgnyteDesktopApp/3.15.5.123/Egnyte.EgnyteDesktopApp.installer.yaml

```diff
@@ -0,0 +1,26 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: Egnyte.EgnyteDesktopApp
+PackageVersion: 3.15.5.123
+Platform:
+- Windows.Desktop
+MinimumOSVersion: 10.0.0.0
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+InstallerSwitches:
+  Silent: /quiet /norestart
+  SilentWithProgress: /passive /norestart

```

Comment:
```suggestion
```
These switches are already passed to the Installer as the InstallerType is set to `wix`.

https://github.com/microsoft/winget-cli/blob/c3b8d5412d90659277ab3ca6e324226b40c4679a/src/AppInstallerCommonCore/Manifest/ManifestCommon.cpp#L571